### PR TITLE
docs: add docker guide, and improve index

### PR DESCRIPTION
**Pull request by pataar**
This adds a guide for installing Mainsail through Docker, and improves the `index.md` just like the `README.md` from the repository.﻿

### CODE REVIEW

**Review by lixxbox — APPROVED** [review]
LGTM

**Review by prcutler — CHANGES_REQUESTED** [review]
Hi, 

This looks great! Two tiny things in docker.md that I'd recommend:

Line 11: nginx should be lower case
Line 21: Remove the word "easily"  - if they're reading the docs, they might not be finding it easy at that moment in time.

That's it - this is excellent!

**Comment by lixxbox** [comment]
I would stick with "NGINX". That's how it's spelled on their website https://www.nginx.com and that's how i included it in the spell checker and changed it in the rest of the docs.

File: index.md [hunk 1]
```diff
@@ -13,28 +13,33 @@ Mainsail makes Klipper more accessible by adding a lightweight, responsive web u
...
+- **Job Queue:** _Queue multiple jobs and add them directly from your slicer_
+- **[Temperature Presets](/features/presets):** _Manage different temperature presets for easy preheating_
+- **[Bed Mesh Visualisation](/features/bedmesh):** _View your bed using a 3D mesh graph_
+- **G-Code Viewer:** _View a 3D render of your print and follow the progress_
+- **Multi-Webcam Support:** _View your print from different angles with multiple webcams_
+- **Timelapse Integration:** _Automatically record a timelapse of your print using [moonraker-timelapse](https://github.com/mainsail-crew/moonraker-timelapse)_
+- **Power Control:** _Control power devices such as relays, TP-Link and Tasmota devices, and more_
+- **Powerful Macro-Management:** _Manage your macros on a micro level_
+- **[Configurable Dashboard](/features/dashboard-organisation):** _Create your own personal dashboard_
+- **[Theming Support](/features/theming/):** _Customizable user interface including logos, backgrounds, and custom CSS_
```
Comment (lixxbox) [inline index.md:40]: Link should point to „/features/theming“ without „/„ at the end.

**Review by lixxbox — CHANGES_REQUESTED** [review]

**Comment by pataar** [comment]
@lixxbox @prcutler Feedback fixed!

**Review by lixxbox — APPROVED** [review]

**Pull request merged**